src/styles/calendar-block.scss

```diff
@@ -0,0 +1,60 @@
+$border-color: #000;
+$day-color:#eee;
+$day-active-color: #ecb9da;
+$font-size: 30px;
+$day-cell: 100px;
+$cell-gap: 1px;
+$default-padding: 10px;
+$calendar-width: 706px;
+$date-position: ($day-cell - $font-size) / 2;
+$weekdays: mon, tue, wed, thu, fri, sat, sun;
+
+.calendar {
+  width: $calendar-width;
+  padding: $default-padding;
+  gap: $cell-gap;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    display: block;
+    position: relative;
+    width: $day-cell;
+    height: $day-cell;
+    border: 1px solid $border-color;
+    border-radius: 5px;
+    background-color: $day-color;
+    transition: transform 500ms;
+    cursor: pointer;
+
+    &::before {
+      counter-increment: day-number;
+      content: counter(day-number);
```

Comment:
You can also iterate from 1 to 31 to add the content using pseudoclass nth-child
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

Reply:
I wanted to try to do it differently, also not sure how to do it with `:nth-child` properly